test/math/ExpDecayMath.ts

```diff
@@ -21,7 +21,7 @@ describe('ExpDecayMath', () => {
     const calcExpDecay = (ethAmount: BigNumberish, timeElapsed: number, halfLife: number) => {
         it(`calcExpDecay(${ethAmount}, ${timeElapsed}, ${halfLife})`, async () => {
             const f = new Decimal(timeElapsed).div(halfLife);
-            if (f.lt(EXP2_INPUT_TOO_HIGH)) {
+            if (f.lte(MAX)) {
```

Comment:
Not aligned with latest changes in the ExpDecay PR

Reply:
merged with the latest changes